### PR TITLE
Add @types/cordova-plugin-save-dialog

### PR DESCRIPTION
**Pull request by dodomorandi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

---

I have not been able to use `dts-gen --dt` in order to generate the `index.d.ts`, but I think it should be fine anyway.

CC @baldoalessandro

### CODE REVIEW

File: types/cordova-plugin-save-dialog/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
+        "@definitelytyped/dt-header": "off",
+        "@typescript-eslint/prefer-namespace": "off"
```
Comment (jakebailey) [inline types/cordova-plugin-save-dialog/.eslintrc.json:4]: These should most definitely not be disabled.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/cordova-plugin-save-dialog/.eslintrc.json (hunk 1 above)
Comment (dodomorandi) [reply types/cordova-plugin-save-dialog/.eslintrc.json]: Ok, probably using `cordova-plugin-device-name` as a template has not been a good idea :laughing:

Comment (jakebailey) [reply types/cordova-plugin-save-dialog/.eslintrc.json]: Wow, I have absolutely no idea why so many packages have this rule disabled. Definitely not right.

**Review by jakebailey — APPROVED** [review]

**Comment by dodomorandi** [comment]
Ready to merge

**Pull request merged**